big_scape/main.py

```diff
@@ -408,13 +454,14 @@ def align_callback(tasks_done: int):
     # query
 
     if run["query_bgc_path"]:
-        query_bin = bs_comparison.ConnectedComponentPairGenerator(
-            query_connected_component, "Query"
-        )
-        query_bin.add_records(
-            [record for record in all_bgc_records if record is not None]
-        )
+        # TODO: select largest cc once, instead of rebuilding bins?
         for cutoff in run["gcf_cutoffs"]:
+            query_bin = bs_comparison.ConnectedComponentPairGenerator(
+                cc_cutoff[cutoff], "Query"
+            )
+            query_bin.add_records(
```

Comment:
could add only the records from the query_connected_component?

Reply:
within add_records() only records in the cc are added